src/main/java/net/minecraftforge/fml/common/AutomaticEventSubscriber.java

```diff
@@ -85,7 +85,7 @@ public static void inject(ModContainer mod, ASMDataTable data, Side side)
             }
             catch (Throwable e)
             {
-                FMLLog.log.error("An error occurred trying to load an EventBusSubscriber {} for modid {}", mod.getModId(), e);
+                FMLLog.log.error("An error occurred trying to load an EventBusSubscriber {} for modid {}", targ.getClassName(), mod.getModId(), e);
```

Comment:
You're not swallowing the error information (e), and not displaying it at all?

Surely you want to have another {} in the log to log out that third parameter (i'm assuming it doesn't have that much info in it, but you should be logging everything passed to the log.error() function

Reply:
Log4j will log a trailing exception parameter.

Reply:
I stand corrected

Magic parameters, probably the reason this slipped in in the first place!

feel free to ignore me then